ivc/src/expr_eval.rs

```diff
@@ -17,6 +19,21 @@ use kimchi::{
 use kimchi_msm::columns::Column as GenericColumn;
 use strum::EnumCount;
 
+#[derive(Clone)]
+/// Generic structure containing column vectors.
+pub struct GenericVecStructure<G: KimchiCurve>(pub Vec<Vec<G::ScalarField>>);
+
+impl<G: KimchiCurve> Index<GenericColumn> for GenericVecStructure<G> {
```

Comment:
This change was in a different PR... maybe I fucked something up with the merge order :(